integr8ly/integreatly-operator
# Small improvement to LDAP test case

# Description


Small improvements to LDAP test case.

**Verification Steps**
Eye review only.
You can also go to OCM UI and try to create a cluster (no need to actually create it, just see the form)
